### DISCUSSION TITLE
You exceeded your current quota, please check your plan and billing details.

### DISCUSSION THREAD

**Discussion by EdgeSignal** [nextai-translator/nextai-translator#938] [General]
![微信截图_20230809134413](https://github.com/openai-translator/openai-translator/assets/5965207/0d14fd3e-2bcf-4232-b389-aedf81e52058)
API Key重新生成也无效，该如何解决这个问题呢？

**Comment by fool-cloud** [comment]
我也遇到这个问题了已经两三个月了，也不知道怎么解决？

**Comment by EdgeSignal** [comment]
1. 没有翻墙
2. 现在的API Key要付费了，可以淘宝购买Key
